docs/webapp/settings/webapp_settings_admin_vaults.md

```diff
@@ -23,24 +28,43 @@ The following configuration values are machine and/or agent specific, so they ca
 * `agent.debug`
 :::
 
-**To create a vault:**
+**To create a Client configuration vault:**
 1. Click **+ Add Vault**
 1. Fill in vault details:
    1. Vault name - Name that appears in the Administrator Vaults table
    1. User Group - Specify the User Group that the vault affects
-   1. Format - Specify the configuration format: HOCON / JSON / YAML.
-1. Fill in the configuration values (click <img src="/docs/latest/icons/ico-info.svg" alt="Info" className="icon size-md space-sm" /> 
-to view configuration file reference). To import and existing configuration file, click <img src="/docs/latest/icons/ico-import.svg" alt="Import" className="icon size-md space-sm" />. 
+   1. Target - Vault type. Select `Client (Agent/SDK/UI)`
+   1. Format - Specify the configuration format: HOCON / JSON / YAML. 
+   1. Fill in the configuration values (click <img src="/docs/latest/icons/ico-info.svg" alt="Info" className="icon size-md space-sm" /> 
+   to view configuration file reference). To import and existing configuration file, click <img src="/docs/latest/icons/ico-import.svg" alt="Import" className="icon size-md space-sm" />. 
 1. Click **Save** 
 
+To apply its contents, a vault should be enabled. 
```

Comment:
@pollfly This is not specific to Client vaults. Better to have this important note as part of the feature overview at the beginning as part of the feature overview.